rizzino_a06b.js

```diff
@@ -0,0 +1,55 @@
+/* Julie Rizzino
+rizzino_a06b.js
+INFO 2124
+Thoendel
+1/23/20 */
+const standardInput = process.stdin;
+standardInput.setEncoding('utf-8');
+
+const story = 
+    `You're walking along Dodge Street minding your own business
+and your suddenly run into the Wizard of Benson standing
+in on the corner of 50th and Dodge.
+
+Do you (C)ontinue walking? (S)top and ask for some magical help?
+(T)rip and fall into a pot hole?
+`;
+
+clearScreen();
+console.log(story);
+
+standardInput.on('data', function(text) {
+    //This line will remove line breaks 
+    //\n\r in window, \r in macOS
+    text = text.replace('\n', '').replace('\r', '');
+    /* WRITE YOUR SOLUTION BETWEEN THIS LINE */
+    clearScreen();
+    //This function will clear the screen.
+    if (text == `C`|| text == `c`) {
```

Comment:
Nice logical condition! Typically - we would use single or double quotes on the C or c - rather than ticks.